src/material/datepicker/datepicker.md

```diff
@@ -566,6 +566,10 @@ The `MatDatepicker` pop-up uses the `role="dialog"` interaction pattern. This di
 multiple controls, the most prominent being the calendar itself. This calendar implements the
 `role="grid"` interaction pattern.
 
+The `matDatepickerCancel` and `matDatepickerApply` provide both visual and audio confirmation of the
+selected date when closing the pop-up. Always enable the confirmation action buttons (see
+[_Confirmation action buttons_](#confirmation-action-buttons)).
```

Comment:
I would swap the order of sentences here (plus small wording change)
```suggestion
Always enable [_confirmation action buttons_](#confirmation-action-buttons). This allows assistive technology users to explicitly confirm their selection before committing a value.
```

Reply:
I like that, it makes it really clear what the instructions are for a11y best practices, and the confirmation-actions-buttons section can go into more detail about the cancel/apply buttons.